tests/zerobase/zerobase.edl

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+enclave {
+    from "openenclave/edl/logging.edl" import oe_write_ocall;
+    from "openenclave/edl/fcntl.edl" import *;
+#ifdef OE_SGX
```

Comment:
can remove this check, and line 9-11

Reply:
...only if the test is renamed or moved down under an sgx directory